tests/unit/test_hookutils.py

```diff
@@ -329,7 +329,9 @@ def _sort(sequence):
         sorted_list = sorted(list(sequence))
         return tuple(sorted_list)
 
-    # Add path with 'hookutils_files' module to ``sys.path``, so tests can find this module - useful for subprocesses.
+    # Add path with 'hookutils_files' module to ``sys.path`` (so analysis in the main process can find it),
+    # and to ``pathex`` (so subprocess-isolated code can find it).
+    monkeypatch.setattr('PyInstaller.config.CONF', {'pathex': [TEST_MOD_PATH]})
     monkeypatch.syspath_prepend(TEST_MOD_PATH)
```

Comment:
Doesn't that mean that this `syspath_prepend()` line can be removed then?

Reply:
Or is `sys.path` needed for non subprocesses?

Reply:
I left it in just in case it might be required by main process, but now I'm not sure it is actually required anywhere in the test. The main idea was to have `data_lists` fixture do the same kind of environment modifications as the `mod_list`  from the same file.